addons/centered/src/html.js

```diff
@@ -0,0 +1,39 @@
+import { document, Node } from 'global';
+
+const style = {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'auto',
+};
+
+const innerStyle = {
+  margin: 'auto',
+};
+
+export default function(storyFn) {
+  const inner = document.createElement('div');
```

Comment:
We could use ids and avoid creating wrappers on each render

Reply:
Where should this wrapper be then? You mean to place it somewhere in DOM?

Reply:
In the same plase as it is now, just with an id

Reply:
Ah, it's useful when navigating between stories with the same decorator. Will do.

Reply:
Yes, or when some decorator like knobs triggers a re-render